### PR TITLE
[major_refactor] Move Devel::AssertOS outside of Dist::Zilla vision

### PR DESCRIPTION
**Pull request by atoomic**
This is fixing Continuous Integration smokers

### CODE REVIEW

**Comment by atoomic** [comment]
still testing, please wait before merging

**Comment by atoomic** [comment]
good to merge :-)

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -43,8 +43,10 @@ jobs:
       - uses: actions/checkout@v1
       - name: perl -V
         run: perl -V
+      - name: Install Dependencies CI
+        run: cpm install -g --show-build-log-on-failure --cpanfile cpanfile.ci
       - name: Install Dependencies
-        run: curl -sL https://git.io/cpm | perl - install -g --show-build-log-on-failure
+        run: cpm install -g --show-build-log-on-failure --cpanfile cpanfile
```
Comment (atoomic) [inline .github/workflows/linux.yml]: note that the linux docker images comes with cpm pre installed :-)

**Comment by exodist** [comment]
Merged to master

**Pull request closed without merging**